blips/blip-0001.mediawiki

```diff
@@ -0,0 +1,236 @@
+
+The bLIP author may update the draft as necessary in the git repository. Updates to drafts should also be submitted by the author as pull requests.
+
+====Transferring bLIP Ownership====
+
+It occasionally becomes necessary to transfer ownership of bLIPs to a new champion. In general, we'd like to retain the original author as a co-author of the transferred bLIP, but that's really up to the original author. A good reason to transfer ownership is because the original author no longer has the time or interest in updating it or following through with the bLIP process, or has fallen off the face of the 'net (i.e. is unreachable or not responding to email). A bad reason to transfer ownership is because you don't agree with the direction of the bLIP. We try to build consensus around a bLIP, but if that's not possible, you can always submit a competing bLIP.
+
+If you are interested in assuming ownership of a bLIP, send a message asking to take over, addressed to both the original author and the bLIP editor. If the original author doesn't respond to email in a timely manner, the bLIP editor will make a unilateral decision (it's not like such decisions can't be reversed).
+
+====bLIP Editors====
+
+The current BOLT editors are the bLIP editors.
+
+====bLIP Editor Responsibilities & Workflow====
+
+For each new bLIP submission, the editors do the following:
+
+* Read the bLIP to check if it is ready: sound and complete. The ideas must make technical sense, even if they don't seem likely to get to final status.
+* The title should accurately describe the content.
+* The bLIP draft must have been sent to the lightning-dev mailing list for discussion.
+* Motivation and backward compatibility (when applicable) must be addressed.
+* Licensing terms must be acceptable for bLIPs.
+
+If the bLIP isn't ready, the editor will send it back to the author for revision, with specific instructions.
+
+Once the bLIP is ready for the repository, the bLIP editor will:
+
+* Assign a bLIP number (generally the PR number or, if preferred by the author, the Issue # if there was discussion in the Issues section of this repository about this bLIP)
+* Merge the corresponding pull request
+* Send a message back to the bLIP author with the next steps.
+
+The bLIP editors are intended to fulfill administrative and editorial responsibilities. They do not pass judgement on bLIPs. The bLIP editors monitor bLIP changes, and update bLIP headers as appropriate.
+
+==What belongs in a successful bLIP?==
+
+bLIPs should be written in mediawiki format.
```

Comment:
I don't feel very strongly about that, but since we use markdown everywhere for the BOLTs, I'd find it more consistent to use markdown for bLIPs as well.

Reply:
Another area I should've done more research on before making the proposal, didn't realize BIPs were mediawiki while BOLTs were markdown 😅 pretty easy fix if that's a general preference. Looking forward to chatting about it today, thanks for your comments!